src/Models/Services/Utils.ts

```diff
@@ -999,3 +1000,18 @@ export function capitalizeFirstLetter(str: string) {
         return str;
     }
 }
+
+/**
+ * Validate if URL is Explorer for CORS enabling
+ */
+export function validateExplorerOrigin(origin: string) {
+    if (
```

Comment:
could we return directly here `return [ADT_ALLOW_LISTED.DEV, ADT_ALLOW_LISTED_URLS.PROD].includes(origin);`